cmd/gapit/screenshot.go

```diff
@@ -75,6 +76,11 @@ func (verb *screenshotVerb) Run(ctx context.Context, flags flag.FlagSet) error {
 		for _, at := range verb.At {
 			commands = append(commands, capture.Command(at[0], at[1:]...))
 		}
+	} else if verb.ExecutedDraws > 0 {
+		commands, err = verb.executedDrawCommands(ctx, capture, client, verb.ExecutedDraws)
```

Comment:
Check that `verb.ExecutedDraws > 0`, return an error otherwise

Reply:
Woops, you do check this on the line above, I read too fast. We could return an error when verb.ExecutedDraws < 0, but it's OK to be silent here since this is a developer-oriented option.